### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by Mahboobr**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+      </div>
+      <div id="ball">
+        <!-- <h1>asdas</h1> -->
+      </div>
+      <div id="paddle">
+        <!-- <h1>sdasda</h1> -->
+      </div>
+    </div>
+  </body>
+  <!-- <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script> -->
```
Comment (JSChilds) [inline index.html]: Don't leave commented-out code in the final product.

File: css/style.css [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+  margin-top: 440px;
+  margin-bottom: auto;
+  position: absolute;
+  background: rgb(255,255,255);
+  border-radius: 10px;
+  width: 20px;
+  height: 20px;
+}
+
+ /* IF ball posX < boards right hand side and greater than left hand side, move the ball in one direction
```
Comment (JSChilds) [inline css/style.css]: Don't leave pseudo-code in the final product.

File: app.js [hunk 3]
```diff
@@ -0,0 +1,183 @@
...
+    console.log(Xarr);
+    console.log(Yarr);
+
+    Yarr.push(posY);
+    var Yans = Yarr[1] - Yarr[0];
+    console.log("Y " + Yans);
+    Yarr.shift();
+
+    console.log(posX);
+
```
Comment (JSChilds) [inline app.js]: This code is very long and not easy to understand.

File: app.js [hunk 4]
```diff
@@ -0,0 +1,183 @@
...
+  var Br84on = 1;  // brick 84 flag
+  var Xarr = [1, 3]; // array for x pos's
+  var Yarr = [1, 3];
+  var LRFLag = 1;
+
+  function frame() {
+
+    Xarr.push(posX);
+    var Xans = Xarr[1] - Xarr[0];
+    console.log("X " + Xans);
```
Comment (JSChilds) [inline app.js]: Don't leave console.logs in the final product.

**Review by JSChilds — CHANGES_REQUESTED** [review]
Good start and the layout looks nice, you just need to work out the logic of how to move the ball around and add in the win and loss conditions. Also put in some instructions to make it clear how to play and clean up your code where possible.
